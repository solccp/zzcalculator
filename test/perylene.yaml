---
structure_id: 1
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 15]
- [3, 10]
- [16, 19]
- [7, 11]
- [12, 13]
- [8, 14]
- [9, 18]
- [17, 20]
---
structure_id: 2
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 15]
- [3, 10]
- [16, 19]
- [7, 11]
- [12, 13]
- [8, 9]
- [14, 17]
- [18, 20]
---
structure_id: 3
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 15]
- [3, 10]
- [16, 19]
- [7, 11]
- [12, 13]
rings:
- [8, 9, 18, 20, 17, 14]
---
structure_id: 4
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 15]
- [3, 10]
- [16, 19]
- [7, 8]
- [9, 18]
- [11, 12]
- [13, 14]
- [17, 20]
---
structure_id: 5
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 15]
- [3, 10]
- [16, 19]
- [9, 18]
- [17, 20]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 6
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 3]
- [15, 19]
- [16, 10]
- [7, 11]
- [12, 13]
- [8, 14]
- [9, 18]
- [17, 20]
---
structure_id: 7
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 3]
- [15, 19]
- [16, 10]
- [7, 11]
- [12, 13]
- [8, 9]
- [14, 17]
- [18, 20]
---
structure_id: 8
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 3]
- [15, 19]
- [16, 10]
- [7, 11]
- [12, 13]
rings:
- [8, 9, 18, 20, 17, 14]
---
structure_id: 9
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 3]
- [15, 19]
- [16, 10]
- [7, 8]
- [9, 18]
- [11, 12]
- [13, 14]
- [17, 20]
---
structure_id: 10
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [2, 3]
- [15, 19]
- [16, 10]
- [9, 18]
- [17, 20]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 11
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [7, 11]
- [12, 13]
- [8, 14]
- [9, 18]
- [17, 20]
rings:
- [2, 3, 10, 16, 19, 15]
---
structure_id: 12
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [7, 11]
- [12, 13]
- [8, 9]
- [14, 17]
- [18, 20]
rings:
- [2, 3, 10, 16, 19, 15]
---
structure_id: 13
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [7, 11]
- [12, 13]
rings:
- [2, 3, 10, 16, 19, 15]
- [8, 9, 18, 20, 17, 14]
---
structure_id: 14
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [7, 8]
- [9, 18]
- [11, 12]
- [13, 14]
- [17, 20]
rings:
- [2, 3, 10, 16, 19, 15]
---
structure_id: 15
number_of_atoms: 20
double_bonds:
- [1, 6]
- [5, 4]
- [9, 18]
- [17, 20]
rings:
- [2, 3, 10, 16, 19, 15]
- [7, 8, 14, 13, 12, 11]
---
structure_id: 16
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 19]
- [16, 10]
- [3, 4]
- [7, 11]
- [12, 13]
- [8, 14]
- [9, 18]
- [17, 20]
---
structure_id: 17
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 19]
- [16, 10]
- [3, 4]
- [7, 11]
- [12, 13]
- [8, 9]
- [14, 17]
- [18, 20]
---
structure_id: 18
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 19]
- [16, 10]
- [3, 4]
- [7, 11]
- [12, 13]
rings:
- [8, 9, 18, 20, 17, 14]
---
structure_id: 19
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 19]
- [16, 10]
- [3, 4]
- [7, 8]
- [9, 18]
- [11, 12]
- [13, 14]
- [17, 20]
---
structure_id: 20
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 19]
- [16, 10]
- [3, 4]
- [9, 18]
- [17, 20]
rings:
- [7, 8, 14, 13, 12, 11]
---
structure_id: 21
number_of_atoms: 20
double_bonds:
- [15, 19]
- [16, 10]
- [7, 11]
- [12, 13]
- [8, 14]
- [9, 18]
- [17, 20]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 22
number_of_atoms: 20
double_bonds:
- [15, 19]
- [16, 10]
- [7, 11]
- [12, 13]
- [8, 9]
- [14, 17]
- [18, 20]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 23
number_of_atoms: 20
double_bonds:
- [15, 19]
- [16, 10]
- [7, 11]
- [12, 13]
rings:
- [1, 2, 3, 4, 5, 6]
- [8, 9, 18, 20, 17, 14]
---
structure_id: 24
number_of_atoms: 20
double_bonds:
- [15, 19]
- [16, 10]
- [7, 8]
- [9, 18]
- [11, 12]
- [13, 14]
- [17, 20]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 25
number_of_atoms: 20
double_bonds:
- [15, 19]
- [16, 10]
- [9, 18]
- [17, 20]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
