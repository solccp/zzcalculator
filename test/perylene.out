 9 + 12 x + 4 x^2
 total: 25
